src/main/java/uk/gov/hmcts/reform/blobrouter/data/reconciliation/statements/SupplierStatementRepository.java

```diff
@@ -64,10 +64,10 @@ public UUID save(NewEnvelopeSupplierStatement statement) throws SQLException {
         }
     }
 
-    public Optional<EnvelopeSupplierStatement> findLatest(LocalDate date) {
+    public Optional<EnvelopeSupplierStatement> findLatestByDate(LocalDate date) {
         try {
             EnvelopeSupplierStatement statement = jdbcTemplate.queryForObject(
-                "SELECT * FROM envelope_supplier_statements WHERE DATE(created_at) = :date "
+                "SELECT * FROM envelope_supplier_statements WHERE date = :date "
```

Comment:
The query changed completely, but tests did not. Looks like we need an extra test?

Reply:
change the date in test, it ws using the same date.